sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 2.5.0-beta.1 (Unreleased)
+## 2.5.0 (2022-03-28)
+This release is compatible with Spring Boot 2.5.0-2.5.11, 2.6.0-2.6.5.
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```

Comment:
And `spring-cloud-dependencies:2021.0.1`.
Refs: https://github.com/Azure/azure-sdk-for-java/pull/27147/files